frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/simulated-region-overview-behavior-tab.component.html

```diff
@@ -1 +1,83 @@
-<p>behavior-tab works!</p>
+<div class="container">
+    <div class="row align-items-start">
+        <div class="col-sm" style="max-width: 30%">
+            <p
+                *ngIf="simulatedRegion.behaviors.length === 0"
+                class="text-muted font-bold"
+            >
+                Es sind noch keine Verhaltensweisen vorhanden.
+            </p>
+            <div class="mb-3 overflow-auto">
+                <div class="list-group">
+                    <button
+                        *ngFor="let behavior of simulatedRegion.behaviors"
+                        class="list-group-item-action d-flex align-items-center flex-nowrap p-0"
+                        [class.active]="behavior === selectedBehavior"
+                        (click)="onBehaviorSelect(behavior)"
+                        class="list-group-item list-group-item-action"
+                    >
+                        {{ behavior | behaviorToGermanName }}
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div class="col-sm" style="max-width: 70%">
+            <div [ngSwitch]="selectedBehavior?.type">
+                <ng-container *ngSwitchCase="'assignLeaderBehavior'"
+                    ><app-simulated-region-overview-behavior-assign-leader></app-simulated-region-overview-behavior-assign-leader
+                ></ng-container>
+                <ng-container *ngSwitchCase="'treatPatientsBehavior'"
+                    ><app-simulated-region-overview-behavior-treat-patients></app-simulated-region-overview-behavior-treat-patients
+                ></ng-container>
+                <ng-container *ngSwitchCase="'unloadArrivingVehiclesBehavior'"
+                    ><app-simulated-region-overview-behavior-unload-arriving-vehicles></app-simulated-region-overview-behavior-unload-arriving-vehicles
+                ></ng-container>
```

Comment:
I think you can apply structural directives directly to the components.

```suggestion
                <app-simulated-region-overview-behavior-assign-leader *ngSwitchCase="'assignLeaderBehavior'"></app-simulated-region-overview-behavior-assign-leader
            >             
            <app-simulated-region-overview-behavior-treat-patients *ngSwitchCase="'treatPatientsBehavior'"></app-simulated-region-overview-behavior-treat-patients
             >
                 <app-simulated-region-overview-behavior-unload-arriving-vehicles *ngSwitchCase="'unloadArrivingVehiclesBehavior'"></app-simulated-region-overview-behavior-unload-arriving-vehicles
            >
```

Reply:
I think the tutorial I read stated using those containers as a good practice, but I think removing them in this case is fine and makes it easier to read, so I will do it.

Reply:
Which tutorial was this?
The only best practice I know of is to use ng-containers instead of unnecessary html-elements to reduce the number of DOM-elements and potentially simplify some CSS.

Reply:
done

Reply:
I would have to go through my search history from Friday because I can't remember right now and it was something like wrap the things in your directive in a ng container to keep everything clean...

Reply:
But this was for multiple elements and I think for only one element it should do nothing...